tools/integration_tests/operations/operations_test.go

```diff
@@ -169,7 +181,8 @@ func TestMain(m *testing.M) {
 	// Note: We are not testing specifically for implicit-dirs because they are covered as part of the other flags.
 	flagsSet := [][]string{
 		// By default, creating emptyFile is disabled.
-		{"--experimental-enable-json-read=true", "--implicit-dirs=true"}}
+		{"--experimental-enable-json-read=true"},
```

Comment:
Is this intended?

Reply:
Yes, want to include implicit-dir with kernel list cache, while I think it is not relevant for --experimental-enable-json-read